go.mod

```diff
@@ -4,7 +4,7 @@
 
 module k8s.io/kubernetes
 
-go 1.12
+go 1.13
```

Comment:
The `build/root/WORKSPACE` configuration specifies exactly `go_version=1.13.4`. Worth doing the same here?

Reply:
Only major+minor is used, so when running go 1.13.4, you get the following result:
```bash
> go mod init example
go: creating new go.mod: module example
> cat go.mod  
module example

go 1.13
```

So we should stick to `go 1.13`